taca/cli.py

```diff
@@ -0,0 +1,32 @@
+import os
```

Comment:
OK! Gives me a peace of mind :smiley:

(You get to [feel like mitsuhiko](https://github.com/mitsuhiko/click/blob/master/click/__init__.py) for a second...)